app/views/foreman_tasks/tasks/index.html.erb

```diff
@@ -39,4 +39,4 @@ $(document).on('click', ".table-two-pane td.two-pane-link", function(e) {
   <% end %>
 </table>
 <%= page_entries_info @tasks %>
-<%= will_paginate @tasks %>
+<%= will_paginate_with_info @tasks %>
```

Comment:
also, I believe you can pass model: _('Tasks') to change the display (instead of foreman-tasks/tasks)